hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestObjectStoreWithFSO.java

```diff
@@ -515,12 +517,11 @@ public void testListKeysWithNotNormalizedPath() throws Exception {
     byte[] input = new byte[length];
     Arrays.fill(input, (byte)96);
 
-    createKey(ozoneBucket, key1, 10, input);
-    createKey(ozoneBucket, key2, 10, input);
-    createKey(ozoneBucket, key3, 10, input);
+    createAndAssertKeys(ozoneBucket, Collections.singletonList(key1));
+    createAndAssertKeys(ozoneBucket, Collections.singletonList(key2));
+    createAndAssertKeys(ozoneBucket, Collections.singletonList(key3));
```

Comment:
Since `createAndAssertKeys` accepts a list, we don't need to call it 3x with a singleton list.

```suggestion
    createAndAssertKeys(ozoneBucket, Arrays.asList(key1, key2, key3));
```